src/snek5000/operators.py

```diff
@@ -109,6 +110,7 @@ def _complete_params_with_default(params):
             "Ly": 2 * pi,
             "Lz": 2 * pi,
             "boundary": ["P", "P", "W", "W", "P", "P"],
+            "boundary_scalars": [],
```

Comment:
@paugier Can you try and see if this helps?

Reply:
`params.oper.boundary_scalars = ['t', 't', 'I', 'I']`

Reply:
Just to be sure, you mean try with `phill` ?

Reply:
Yes, but only the mesh and SIZE file generation.